tests/e2e/testRunner.js

```diff
@@ -213,8 +213,8 @@ const runTests = async () => {
 
     let progressLog = Logger.progressInfo('Installing apps and reversing port');
 
-    await installApp('android', config.MAIN_APP_PACKAGE, mainAppPath);
-    await installApp('android', config.DELTA_APP_PACKAGE, deltaAppPath);
+    await installApp(config.MAIN_APP_PACKAGE, mainAppPath, 'android');
+    await installApp(config.DELTA_APP_PACKAGE, deltaAppPath, 'android');
```

Comment:
As I see a default value is "android", so we can skip it here.
```suggestion
    await installApp(config.MAIN_APP_PACKAGE, mainAppPath);
    await installApp(config.DELTA_APP_PACKAGE, deltaAppPath);
```